Manual.md

```diff
@@ -761,6 +762,29 @@ versions.  Example: `ignore="*b*"`
 - `version` is the version number used to compare against
 upstream versions. Example: `version=${version//./_}`
 
+<a id="patches"></a>
+### Handling patches
+
+Sometimes software needs to be patched, most commonly to fix bugs that have
+been found or to fix compilation with new software.
+
+To handle this, xbps-src will automatically apply all files ending with .patch
+and .diff that exist in the directory `patches`, on the same directory where the
+package's template is found, using `patch(1)` with `-Np0` during the `do_patch()`
+phase, the variable `PATCHESDIR` is also made available pointing to the `patches`
+directory.
```

Comment:
```
 To handle this, xbps-src has a patching functionality. It will look for all files
that match the glob `srcpkgs/$pkgname/patches/*.{diff,patch}` and will
automatically apply all files it finds using `patch(1)` with `-Np0`. This happens
during the `do_patch()` phase. The variable `PATCHESDIR` is
available in the template, pointing to the `patches` directory.
```